plugins/modules/foreman_role.py

```diff
@@ -72,8 +82,7 @@
     organizations:
       - "Basalt"
     filters:
-      - resource: 'Host'
```

Comment:
This is "interesting". The entry wasn't used before, but it could. When listing permissions, you can pass a `resource_type`. But then again, permission names are unique and there is not really a point in doing so (unless for a nicer grouping/listing in an UI, which we don't have).

Reply:
I'm assuming this means, ACK to this removal. Resolving that conversation.